terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1582,7 +1591,7 @@ def qq(*words):
                                                         # reconfigure logs (name based on UNIX timestamp of when
                                                         # reconfigure initiated). Only the most recent file, by
                                                         # modification time, matching the wildcard is collected.
-                                                        'reconfigure/*.log'
+                                                        'reconfigure/*'
```

Comment:
If the files end in `.log`, the pattern should match on that.

Reply:
https://github.com/DataBiosphere/azul/pull/5187#issuecomment-1564818707